packages/framework/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -2,10 +2,12 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+// eslint-disable-next-line unicorn/prefer-node-protocol
 import { strict as assert } from "assert";
 import { AttachState } from "@fluidframework/container-definitions";
 import { ContainerSchema } from "@fluidframework/fluid-static";
 import { ISharedMap, IValueChanged, SharedMap } from "@fluidframework/map";
+import type { IFluidHandle } from "@fluidframework/core-interfaces";
```

Comment:
This is odd. What is forcing this?

Reply:
See the changes on lines 256-257 of this file:

```ts
        const hdl = await map1.get("new-pair-id") as IFluidHandle;
        const obj = hdl.get();
```

Is there a better way to address the "unsafe any" issues in this case?